.circleci/config.yml

```diff
@@ -250,6 +266,29 @@ commands:
           command: |
             psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
             psql -c 'create database webservice_test with owner = dockstore;' -U postgres
+  old_python: # Need Python 3.6 for old Dockstore 1.7 requirements.txt
+    steps:
+      - run:
+          name: Install & use Python 3.6
+          command: |
+            sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
This is a bit remarkable, only two developers for distributing Python on Ubuntu? 
https://github.com/deadsnakes

Wonder if there is a more popular way, maybe snap?

Reply:
`deadsnakes/ppa` indeed is the most widely referenced way to installing specific Python versions on Ubuntu (when I Google search `python ubuntu`, all the relevant results show this).

For people who frequently switch Python versions & packages, there's `conda` which is certainly overkill here.

Ubuntu already comes with a distribution of Python (3.8), and it's relatively rare for people to *just* want older releases of Python 3, so I don't think it's too strange that only 2 developers is a big surprise.

Reply:
Reminds me of https://linustechtips.com/topic/145130-the-internet-is-being-protected-by-two-guys-named-steve-heartbleed-bug/